Source/ProgressionSystemRuntime/Public/ModuleStructures.h

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#pragma once
+
+#include "Bomber.h"
+#include "Structures/PlayerTag.h"
+#include "ModuleStructures.generated.h"
+
+/**
+ * Basic structure for all save data information regarding the progression.
+ * Same structure will reflected in the save file. Initial load performed based on the data in the DT Table 
+ */
+USTRUCT(BlueprintType)
+struct FProgressionRowData : public FTableRowBase
+{
+	GENERATED_BODY()
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
+	ELevelType Map = ELevelType::None;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
+	FPlayerTag Character = FPlayerTag::None;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
+	TObjectPtr<UTexture> Icon = nullptr;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
+	FColor ActiveColor = FColor::Red;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
+	FColor InactiveColor = FColor::Red;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
+	int32 CurrentLevelProgression = 0;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
+	int32 PointsToUnlock = 0;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
+	bool IsLevelLocked = true;
```

Comment:
I was trying to use; 
NoExport - no applicable for the UPROPERTY
Transient - Ignores Save/Load which is necessary, since we do work with saves. 
empty -  keeps visible
SkipSerialization - also ignore the saves for some reason. 

Agreed with @JanSeliv to keep empty.

Reply:
CurrentLevelProgression also seems to be runtime property, but not for designer. Please also do it to be just regular `UPROPERTY()`

Reply:
done